operations/routingrules.md

```diff
@@ -0,0 +1,380 @@
+
+## Update routing rule
+
+Updates routing rules.
+
+### Request
+
+`[PlatformAddress]/api/connector/v1/routingRules/update`
+
+```javascript
+{
+    "AccessToken": "3BCD0D22F20A4A8BB2A8AF2A00D5D3FC-30FA935326D2715A03E9E57B6A11B42",
+    "ClientToken": "9B6763125AF04F44818BAF2A00D5D3A4-513EDF6A056FFF29D91271C55E8E998",
+    "Client": "Sample Client 1.0.0",
+    "RoutingRules":
+    [
+        {
+            "RoutingRuleId": "ff785b22-5422-4d1d-87f4-af2e00b3dfda",
+            "CompanyId": { "Value": "35f493d1-6cf7-4132-982a-a5540549c322" },
+            "CompanyRelation": { "Value": "PartnerCompany" },
+            "ServiceId": { "Value": "708b3509-69ad-4a92-841b-d81f103edcee" },
+            "Applicability": { "Value": "Always" },
+            "RouteType": { "Value": "AllStayItems" }
+        },
+        {
+            "RoutingRuleId": "d98c9611-0006-4691-a835-af2e00b170c4",
+            "CompanyId": { "Value": "73a1fc8e-11c5-4019-b296-9234b651fc68" },
+            "CompanyRelation": { "Value": "TravelAgency" },
+            "ServiceId": { "Value": "708b3509-69ad-4a92-841b-d81f103edcee" },
+            "Applicability": { "Value": "Always" },
+            "RouteType": { "Value": "SelectedStayItems" },
+            "SelectedStayItems": 
+            {
+                "Nights" : { "Value":  false },
+                "CityTax" : { "Value": true },
+                "ProductCategoryIds": { "Value": [
+                    "004fa262-7b08-4853-b544-af2a00d5cf1f"
+                ]}
+            }
+        },
+
+    ]
+}
+```
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `ClientToken` | string | required | Token identifying the client application. |
+| `AccessToken` | string | required | Access token of the client application. |
+| `Client` | string | required | Name and version of the client application. |
+| `RoutingRules` | array of [Routing rule update parameters](routingrules.md#routing-rule-update-parameters) | required, max 1000 items | Routing rules to be updated. |
+
+#### Routing rule update parameters
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `RoutingRuleId` | string | required | Unique identifier of the routing rule. |
+| `CompanyId` | [String update value](#string-update-value) | optional | Unique identifier of the [Company](companies.md#company) \(or `null` if should not be updated\). |
+| `CompanyRelation` | [String update value](#string-update-value)| optional | [Company relation](routingrules.md#company-relation) to the routing rule \(or `null` if should not be updated\). |
+| `Applicability` | [String update value](#string-update-value) | required | [Applicability](routingrules.md#applicability) that determines if routing rule apply to all future reservations with this company or travel agency attached or only future reservations that are prepaid in online travel agency (OTA) \(or `null` if should not be updated\). |
```

Comment:
Now that I'm thinking, is it correct that `null` should it not be updated? Doesn't providing a null mean nullifying? And not providing not updating? 🤔